<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1176</width>
    <height>721</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Amirkabir University Of Technology</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="lbl_hist_1">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>60</y>
      <width>300</width>
      <height>100</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>CrossCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Input Images :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>291</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Project # 1 : Edwin Babaians [92135015]</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_loadimage">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Image</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_processimage">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Preprocess Image</string>
    </property>
   </widget>
   <widget class="QListWidget" name="lst_names">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>61</y>
      <width>256</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_grayscale">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>480</y>
      <width>117</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>grayscale</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_histogramequalization">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>500</y>
      <width>201</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Histogram Equalization</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_calchitogram">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>540</y>
      <width>201</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Calc Histogram</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>60</y>
      <width>512</width>
      <height>512</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>CrossCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_hist_2">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>160</y>
      <width>300</width>
      <height>100</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>CrossCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_hist_3">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>260</y>
      <width>300</width>
      <height>100</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>CrossCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_ContrastStretching">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>520</y>
      <width>201</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Contrast Stretching</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_gaussiannoise">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>560</y>
      <width>201</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Gaussian Noise</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_saltpapernoise">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>580</y>
      <width>201</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Saltpaper Noise</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>30</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Image :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>30</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Histogram :</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_uniformnoise">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>600</y>
      <width>201</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Uniform Noise</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_filtermedian">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>660</y>
      <width>201</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Median Filter</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_filtergaussian">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>680</y>
      <width>201</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Gaussian Filter</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_filteravg">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>640</y>
      <width>201</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>AVG Filter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_contours">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>660</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Find Contours</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_cany">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>620</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Cany Edge Detection</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_binary">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>580</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Binary Threshold</string>
    </property>
   </widget>
   <widget class="QSlider" name="slider_binary">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>580</y>
      <width>291</width>
      <height>29</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_setp">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>620</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>save step</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_show">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>580</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>show current image</string>
    </property>
   </widget>
   <widget class="QSlider" name="slider_cany1">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>610</y>
      <width>291</width>
      <height>29</height>
     </rect>
    </property>
    <property name="maximum">
     <number>500</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="slider_cany2">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>630</y>
      <width>291</width>
      <height>29</height>
     </rect>
    </property>
    <property name="maximum">
     <number>500</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="txt_count">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>440</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="plainText">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="txt_sizem">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>660</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="plainText">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_contours_2">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>660</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>K-Means</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_contours_3">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>410</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Gabor Filter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_contours_4">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>450</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Gabor set</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
